squidb/src/com/yahoo/squidb/utility/SquidUtilities.java

```diff
@@ -186,7 +186,7 @@ private static void addColumnToRowBuilder(StringBuilder builder, String value, i
      * A version of {@link Collection#addAll(Collection)} that works on varargs without calling Arrays.asList, which is
      * a performance and memory boost
      */
-    public static <T> void addAll(Collection<T> collection, T... objects) {
+    public static <T> void addAll(Collection<T> collection, T[] objects) {
```

Comment:
It's really intended as a wrapper to Collections.addAll, but it does a null check. It's never called directly as a varargs method itself, but instead callers pass their own varargs argument or an array, whichever they happen to have.